npm run browser-build failed

npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\
node_modules\\npm\\bin\\npm-cli.js" "run" "browser-build"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! mqtt@2.1.3 browser-build: `rm -rf dist/; mkdir dist/; browserify mqtt.j
s -s mqtt > dist/mqtt.js; uglifyjs --screw-ie8 < dist/mqtt.js > dist/mqtt.min.js
`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the mqtt@2.1.3 browser-build script 'rm -rf dist/; mkdir dist
/; browserify mqtt.js -s mqtt > dist/mqtt.js; uglifyjs --screw-ie8 < dist/mqtt.j
s > dist/mqtt.min.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the mqtt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rm -rf dist/; mkdir dist/; browserify mqtt.js -s mqtt > dist/mqtt.j
s; uglifyjs --screw-ie8 < dist/mqtt.js > dist/mqtt.min.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs mqtt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls mqtt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\XiaoChutian\node_modules\mqtt\npm-debug.log

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\ch
okidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@
1.0.15: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"
})
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\
node_modules\\npm\\bin\\npm-cli.js" "install" "."
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! mqtt@2.1.3 prepublish: `npm run browser-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the mqtt@2.1.3 prepublish script 'npm run browser-build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the mqtt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run browser-build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs mqtt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls mqtt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\XiaoChutian\node_modules\mqtt\npm-debug.log

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "test": "mocha --bail",
     "pretest": "standard",
     "prepublish": "npm run browser-build",
-    "browser-build": "rm -rf dist/; mkdir dist/; browserify mqtt.js -s mqtt > dist/mqtt.js; uglifyjs --screw-ie8 < dist/mqtt.js > dist/mqtt.min.js",
+    "browser-build": "rimraf dist/ && mkdirp dist/ && browserify mqtt.js -s mqtt > dist/mqtt.js && uglifyjs --screw-ie8 < dist/mqtt.js > dist/mqtt.min.js",
     "browser-test": "zuul --server test/browser/server.js --local --open test/browser/test.js",
     "sauce-test": "zuul --server test/browser/server.js --tunnel ngrok -- test/browser/test.js"
   },
@@ -71,8 +71,10 @@
   "devDependencies": {
     "browserify": "^13.1.1",
     "mocha": "*",
+    "mkdirp": "^0.5.1",
     "mqtt-connection": "^3.0.0",
     "pre-commit": "^1.1.3",
+    "rimraf": "^2.5.4",
     "should": "*",
     "sinon": "~1.17.0",
     "standard": "^8.0.0",

